internal/pkg/mount/switchroot/switchroot.go

```diff
@@ -72,6 +73,10 @@ func Switch(prefix string, mountpoints mount.Points) (err error) {
 		return fmt.Errorf("error deleting initramfs: %w", err)
 	}
 
+	if err := selinux.Init(); err != nil {
+		return err
+	}
```

Comment:
We could potentially initialize it in first stages of machined as well. Let's consider pros and cons

Reply:
As we use `typetransition` now, then not, because we need machined to already be in the correct context.